src/controller/TestController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\controller;
+
+class TestController
+{
+
+
+    /**
+     * Summary of index
+     * this function is temporary, just to test
+     * 
+     * @param mixed $id
```

Comment:
Attention maintenant on est en PHP 8 et tu fais de la programmation Objet (POO). Donc il faut TOUJOURS typer tes variables et tes fonctions.
```suggestion
     * @param int $id
```

Reply:
je m'en suis aperçue c'est ce que j'étais en train de faire